auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -46,7 +46,7 @@
                 style="@style/FirebaseUI.EditText"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:inputType="textPersonName"
+                android:inputType="textCapWords"
```

Comment:
`textPersonName` wasn't working for me for some reason and it doesn't work in my app either so I'm not sure what's going on. `textCapWords` replicates the behavior we actually want so it should be fine.